ElectroWeakAnalysis/ZEE/interface/AnalysisErsatz.h

```diff
@@ -16,6 +16,8 @@
 //
 //
 
+#ifndef AnalysisErsatz_h
```

Comment:
hi @Teemperor - please include the subsystem and package name in the include guard. Thanks